<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IM Client - Chat</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_main">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="left_panel">
      <layout class="QVBoxLayout" name="verticalLayout_left">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_top">
         <item>
          <widget class="QLabel" name="label_target">
           <property name="text">
            <string>Send to:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_target"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_status">
           <property name="text">
            <string>Disconnected</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser_messages"/>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_input"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_actions">
         <item>
          <widget class="QToolButton" name="btn_file">
           <property name="text">
            <string>File</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_send">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget_side">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_transfers">
       <attribute name="title">
        <string>Transfers</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_transfers">
        <item>
         <widget class="QTableWidget" name="table_transfers"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_online">
       <attribute name="title">
        <string>Online</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_online">
        <item>
         <widget class="QListWidget" name="list_online"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
